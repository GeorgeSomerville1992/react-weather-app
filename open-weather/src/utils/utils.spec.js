import helper from './helpers';

const fixture = {
  list: [
    {
      dt: 1531774800,
      main: {
        temp : 293.05
      }
    },
    {
      dt: 1531785600,
      main: {
        temp : 293.05
      }
    },
    {
      dt: 1531796400,
      main: {
        temp : 293.05
      }
    },
  ]
}

describe('Basic Utils tests', () => {
  it('FilterData', () => {
    const bucket = helper.getNextFiveDays()

    expect(helper.filterData(bucket, fixture)).toMatchSnapshot()
  })
})