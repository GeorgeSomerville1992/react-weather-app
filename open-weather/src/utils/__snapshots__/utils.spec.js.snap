// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic Utils tests FilterData 1`] = `
Array [
  Object {
    "currentDay": "Tuesday",
    "times": Array [
      Object {
        "dt": 1531785600,
        "main": Object {
          "temp": 293.05,
        },
      },
      Object {
        "dt": 1531796400,
        "main": Object {
          "temp": 293.05,
        },
      },
    ],
  },
  Object {
    "currentDay": "Wednesday",
    "times": Array [],
  },
  Object {
    "currentDay": "Thursday",
    "times": Array [],
  },
  Object {
    "currentDay": "Friday",
    "times": Array [],
  },
  Object {
    "currentDay": "Saturday",
    "times": Array [],
  },
  Object {
    "currentDay": "Sunday",
    "times": Array [],
  },
]
`;
